app/controllers/nonsipp/receivetransfer/RemoveTransferInController.scala

```diff
@@ -94,6 +94,7 @@ class RemoveTransferInController @Inject()(
                       .removePages(
                         transferInPages(srn, memberIndex, index)
                       )
+                      .remove(ReceiveTransferProgress(srn, memberIndex, index))
```

Comment:
Since ReceiveTransferProgress is also included in the transferInPages list, this line can be deleted - I think this was my mistake when we were pairing, as I didn't realise this was already being removed elsewhere.